Dockerfile

```diff
@@ -21,7 +23,7 @@ RUN rm -rf ${ANDROID_DIR}/tools \
  && rm -f tools.zip
 
 # Install SDK tools
-# `yes` is required to accept licenses:
+# Yes is required to accept licenses
```

Comment:
`yes` references the CLI utility, so why change this?

Reply:
Markdown is useless in shell scripts so there's no reason to wrap the word.

Reply:
I wouldn't hold up the PR for it, but this change makes things more ambiguous imo.

Reply:
I could just remove the line entirely, the link directly below explains it perfectly.